File: Rdpzsd.Models/Dtos/Rdpzsd/Parts/PersonStudentDuplicateDiploma/PersonStudentDuplicateDiplomaCreateDto.cs
using Rdpzsd.Models.Dtos.Rdpzsd.Parts.PersonStudentSticker;
using Rdpzsd.Models.Models.Rdpzsd.Parts;

namespace Rdpzsd.Models.Dtos.Rdpzsd.Parts
{
    public class PersonStudentDuplicateDiplomaCreateDto
    {
        public StickerDto StickerDto { get; set; }
        public PersonStudentDuplicateDiploma NewDuplicateDiploma { get; set; }
    }
}

File: WebApplication/src/rdpzsd/dtos/parts/person-student-duplicate-diploma/person-student-duplicate-diploma-create.dto.ts
import { PersonStudentDuplicateDiploma } from "src/rdpzsd/models/parts/person-student/person-student-duplicate-diploma.model";
import { StickerDto } from "../person-student-sticker/sticker.dto";

export class PersonStudentDuplicateDiplomaCreateDto {
    stickerDto: StickerDto;
    newDuplicateDiploma: PersonStudentDuplicateDiploma;
}